inductiva/tasks/task.py

```diff
@@ -370,6 +377,17 @@ def get_output_info(self) -> output_info.TaskOutputInfo:
             files=output_files,
         )
 
+    def _contains_only_std_files(self, output_dir: types.Path) -> bool:
+        """Check if the output archive contains only stdout and stderr files.
+
+        Returns:
+            True if the output archive contains only stdout and stderr files,
+            False otherwise.
+        """
+        output_files = list(pathlib.Path(output_dir).iterdir())
+        return all(
+            file.name in self.STANDARD_OUTPUT_FILES for file in output_files)
+
     def download_outputs(
```

Comment:
This method had a refactor that was only merged today. If you need help resolving the conflicts please reach out.